specs/spec_reader_test.go

```diff
@@ -2,46 +2,17 @@ package specs
 
 import (
 	"testing"
-
-	"github.com/stretchr/testify/require"
 )
 
-var sources = map[string]Source{
-	"aws.yml": {
-		Name:        "aws",
-		Path:        "aws",
-		Version:     "v1.0.0",
-		Concurrency: 10,
-		Registry:    RegistryLocal,
-	},
-}
-
-var destinations = map[string]Destination{
-	"postgresql.yml": {
-		Name:      "postgresql",
-		Path:      "postgresql",
-		Version:   "v1.0.0",
-		Registry:  RegistryGrpc,
-		WriteMode: WriteModeOverwrite,
-	},
-}
-
 func TestLoadSpecs(t *testing.T) {
-	specReader, err := NewSpecReader("testdata/valid")
+	specReader, err := NewSpecReader("testdata")
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	require.Equal(t, sources, specReader.sources)
-	require.Equal(t, destinations, specReader.destinations)
-}
-
-func TestWrongKind(t *testing.T) {
-	_, err := NewSpecReader("testdata/wrong_source")
-	require.Equal(t, err.Error(), "failed to unmarshal file invalid.yml: failed to decode json: unknown kind test")
-}
-
-func TestNoSpecs(t *testing.T) {
-	_, err := NewSpecReader("testdata")
-	require.Equal(t, err.Error(), "no valid config files found in directory testdata")
+	if len(specReader.sources) != 1 {
+		t.Fatalf("expected 1 source got %d", len(specReader.sources))
+	}
+	if len(specReader.destinations) != 1 {
+		t.Fatalf("expected 1 destination got %d", len(specReader.destinations))
+	}
```

Comment:
nit: Go convention is to have the test failure order mirror the comparison above it, so it would be `got %d sources, want %d` in this case (See https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures)